handlers/officersHandler.go

```diff
@@ -0,0 +1,32 @@
+package handlers
+
+import (
+	"encoding/json"
+	"github.com/companieshouse/chs-delta-api/models"
+	"net/http"
+)
+
+// OfficerHandler offers a handler by which to publish a message onto a kafka topic.
+type OfficerHandler struct {
+}
+
+// NewOfficerHandler returns a OfficerHandler.
+func NewOfficerHandler() *OfficerHandler {
+	return &OfficerHandler{
+	}
+}
+
+// ServeHTTP accepts an incoming officer request via a POST method and validates and forwards to the XXX service.
+func (kp *OfficerHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
```

Comment:
Is there a reason to call this method ServeHTTP instead of explaining what it does Like HandleOfficerDelta or something similar ?

Reply:
There's no particular reason so can rename

Reply:
This will be discussed and decided in more detail during the auth ticket implementation. See PR description for link to auth ticket.